DXRando/DeusEx/Classes/SmugglerElevatorTracker.uc

```diff
@@ -0,0 +1,49 @@
+class SmugglerElevatorTracker expands FlagToggleTrigger;
+
+var bool street, elevatorMoved;
+
+static function SmugglerElevatorTracker CreateSET(DXRando dxr)
+{
+    local SmugglerElevatorTracker set;
+
+    set = dxr.Spawn(class'SmugglerElevatorTracker',, 'elevatorbutton');
+    set.street = dxr.localURL == "02_NYC_STREET" || dxr.localURL == "04_NYC_STREET" || dxr.localURL == "08_NYC_STREET";
+    set.flagExpiration = dxr.dxInfo.missionNumber + 1;
```

Comment:
Flag expiration could be passed into the function as well

Reply:
Flag expiration is passed (optionally) to the more generic `FlagToggleTrigger` class's Create function. But this is not meant to be a generic class.